packages/faceted-search/src/queryClient/tql.js

```diff
@@ -34,15 +35,17 @@ const prepareBadges = flow([removeBadgesWithEmptyValue, getBadgesQueryValues]);
  * @param {Query} query instance of Query class
  */
 const getTqlClassOperatorsDictionary = query => ({
-	contains: query.contains,
-	containsIgnoreCase: query.containsIgnoreCase,
-	equals: query.equal,
-	in: query.in,
-	notEquals: query.unequal,
-	greaterThan: query.greaterThan,
-	greaterThanOrEquals: query.greaterThanOrEqual,
-	lessThan: query.lessThan,
-	lessThanOrEquals: query.lessThanOrEqual,
+	[operatorNames.contains]: query.contains,
+	[operatorNames.containsIgnoreCase]: query.containsIgnoreCase,
+	[operatorNames.equals]: query.equal,
+	[operatorNames.in]: query.in,
+	[operatorNames.notEquals]: query.unequal,
+	[operatorNames.greaterThan]: query.greaterThan,
+	[operatorNames.greaterThanOrEquals]: query.greaterThanOrEqual,
+	[operatorNames.lessThan]: query.lessThan,
+	[operatorNames.lessThanOrEquals]: query.lessThanOrEqual,
+	[operatorNames.complies]: query.complies,
```

Comment:
Can you add test for `complies` and `wordComplies` in `/src/queryClient/tql.test.js`